src/services/member/plugins/thumbnail/index.ts

```diff
@@ -94,7 +92,7 @@ const plugin: FastifyPluginAsyncTypebox<GraaspThumbnailsOptions> = async (fastif
       if (!url) {
         reply.status(StatusCodes.NO_CONTENT);
       } else {
-        fileService.setHeaders({ reply, replyUrl, url, id: memberId });
+        reply.status(StatusCodes.OK).send(url);
```

Comment:
Same comment here, use the simple `return url` ? Or is it a better practice to use the `send` ?